src/xfer/Transfer.C

```diff
@@ -96,9 +96,9 @@ Transfer::load(const YAML::Node & node)
   realmPairName_.second = realmPair[1].as<std::string>() ;
 
   // set bools for variety of mesh part declarations
-  const bool hasOld = node["mesh_part_pair"];
-  const bool hasNewFrom = node["from_target_name"];
-  const bool hasNewTo = node["to_target_name"];
+  const bool hasOld = node["mesh_part_pair"]?true:false;
```

Comment:
are these changes to handle the latest version of `yaml-cpp`?

Reply:
Yes, I couldn't compile it with the latest version of `yaml-cpp` unless I made those changes.